src/firefly/js/tables/ui/TablePanelOptions.jsx

```diff
@@ -163,7 +164,7 @@ function Options({uiState, tbl_id, tbl_ui_id, ctm_tbl_id, onOptionReset, onChang
                 </div>
             </div>
             <div>
-                {showPaging &&
+                {showPaging && pageSize!==MAX_ROW &&
```

Comment:
so `MAX_ROW` might not be enough. it might need to be something like `pageSize>=number of rows in table`
Many client tables are not using MAX_ROW they are using something big like 10000.
@loitly what do you think.

Reply:
If there's not many places, I would prefer to change them to MAX_ROW.  Using `pageSize >= total row` is not quite the same.  You can have a table with 80 rows while the default pageSize of 100.  In that case, you would still want them to be able to change page size.

Reply:
We can also default pageSize to MAX_ROW when not set for client tables.  I think I may have done this already.  But, if not, it's something we can add.

Reply:
ok, maybe @jaladh-singhal can track them down

Reply:
Ok I'll give it a try